### DISCUSSION TITLE
Query Explain

### DISCUSSION THREAD

**Discussion by k-capehart** [k-capehart/go-salesforce#75] [General]
Follow up from here: https://github.com/k-capehart/go-salesforce/pull/71

This feature, while useful, is still in beta as outlined here: https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_query_explain.htm

With the risk of this functionality changing before leaving beta, I'm inclined to leave it in the feature branch `query-explain` until it has been fully released. After which I'll be happy to merge it in after verifying if any changes are needed.

@TuSKan Let me know your thoughts on that.

**Comment by TuSKan** [comment]
I totally agree to keep this feature out of the branch master, because it still on beta on Salesforce.
I will greatly appreciated if you can keep this branch synced with master.
Thanks for your great work with this package.
@k-capehart
